### PR TITLE
Add Steps

### PR DESCRIPTION
**Pull request by sjreed01**
Added steps for recipie

### CODE REVIEW

File: cookierecipie.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Steps
+1) Make Cookie Dought
+2) Bake
```
Comment (m-osborn) [inline cookierecipie.txt]: What temp?

Comment (sjreed01) [reply cookierecipie.txt]: 350

Comment (m-osborn) [reply cookierecipie.txt]: Works for me!

**Comment by m-osborn** [comment]
Looks good! Approved

**Pull request merged**